### ISSUE TITLE
how do i use this? lol

### ISSUE THREAD

**Issue by RISE7447** [TestForCry/robloxhacks#1]
how do i use this? lol

**Comment by SpicyToastt** [comment]
how dose this work

**Comment by TheBlackSplitYt** [comment]
You get a script executor like krnl then put the code in and press execute

**Comment by jamarion222** [comment]
what do you press?

**Comment by DevidIsCool** [comment]
wait kernel or just krnl and do i have to use cmd prompt im confused also if powershell is support?

**Comment by HelloVeryNiceToMeetYou** [comment]
I think you go on the file, copy and paste then go on that game in roblox. This is only useful for computers but not sure if it works, then you go in the searchbar and type " Javascript: " exactly like that then paste the hack in and hit enter, Again, probably only useful for computers.

**Comment by zpenguin360** [comment]
im curious how this works does it add onto the launcher in anysort of way
also whats the chance of getting banned for this

**Comment by HelloVeryNiceToMeetYou** [comment]
> im curious how this works does it add onto the launcher in anysort of way also whats the chance of getting banned for this

If it's more than a year old, you have chances of getting banned. To be safe, make a new account

**Comment by zpenguin360** [comment]
oh yeah personaly i wouldnt want my old 2007 account that has a lot of limiteds to get banned so yeah

**Comment by zpenguin360** [comment]
yeah by the way you do know you are talking to a mod right now , right?

**Comment by zpenguin360** [comment]
look man i don't know if u actually work on this but if u do care about this friend my alt and we can work out a compromise but if u ignore this the hacks will cease to function 
https://web.roblox.com/users/2229758268/profile
if u dont work on this tell me who does ill contact them

**Comment by zpenguin360** [comment]
ok nvrmind im fixing this shit

**Comment by zpenguin360** [comment]
im wiping this
